Tribler/Test/Core/test_osutils.py

```diff
@@ -61,14 +61,21 @@ def test_fix_filebasename(self):
             fixedname = fix_filebasename(name)
             assert fixedname == name_table[name], (fixedname, name_table[name])
 
+    def test_get_android_api_version(self):
+        if sys.platform in ['win32','cygwin','darwin','os2','os2emx','riscos','atheos']:
+            self.assertIsNone(get_android_api_version())
+        elif 'ANDROID_PRIVATE' in os.environ:
```

Comment:
you can get rid of one branch here by moving the `elif android_private in os.environ` to the first branch, and make the else case assertIsNone